index.html

```diff
@@ -5,6 +5,37 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>My Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
   </head>
-  <body></body>
+  <body>
+    <nav class="navbar">
+      <div class="navbar__logo">Ahmed</div>
+      <div class="navbar__hamburger">
+        <img src="images/Union.svg" alt="menu" />
+      </div>
+    </nav>
+
+    <header class="header">
+      <h1 class="header__title">I’m Ahmed Glad to see you!</h1>
+      <p class="header__text">
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <a href="#" class="header__link">Let's connect</a>
+
+      <div class="header__icons">
+        <img src="images/linkedin.svg" alt="linkedin" />
+        <img src="images/twitter.svg" alt="twitter" />
+        <img src="images/medium.svg" alt="medium" />
+        <img src="images/github.svg" alt="github" />
+      </div>
+    </header>
+  </body>
 </html>
```

Comment:
And lastly, I would encourage you to keenly compare your design with the figma design and try to make them match as close as you can. As you can see the paddings do vary 👇🏻 

| Your Design | Figma Design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/4247740/148264730-f7e210ed-ed56-46ea-9b70-4e46535a090d.png) | ![image](https://user-images.githubusercontent.com/4247740/148265255-0cc09ce1-ebad-4636-8ef4-7d46508418fc.png) |